testem.js

```diff
@@ -8,21 +8,21 @@ module.exports = {
     disable_watching: true,
     reporter: new DotReporter(),
     parallel: 2,
-    launch_in_ci: [
-        'Chrome',
-    ],
-    launch_in_dev: [
-        'Chrome',
-    ],
-    browser_start_timeout: 60,
+    launch_in_ci: ['Chrome'],
+    launch_in_dev: ['Chrome'],
+    browser_start_timeout: 120,
```

Comment:
I think this is straight from the ember blueprints. I am in favor of it because it's just a timeout config but we have not experienced timeout issues with testem connecting to the browser since we switch to github actions.